addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -200,6 +200,8 @@ var WysiwygMultizone = Wysiwyg.extend({
             self.$('[data-oe-readonly]').addClass('o_not_editable').attr('contenteditable', false);
             self.$('.oe_structure').attr('contenteditable', false).addClass('o_fake_not_editable');
             self.$('[data-oe-field][data-oe-type="image"]').attr('contenteditable', false).addClass('o_fake_not_editable');
+
+            self.$('li.o_empty_phone').removeClass('d-none');
```

Comment:
Remove this, does not work and too dependent of the editor being reverted.

Reply:
It is working* but I'll remove that then